### ISSUE TITLE
Errors are silently swallowed in 0.6.1

### ISSUE THREAD

**Issue by ches** [lihaoyi/Ammonite#402]
I seem to have lost all error reporting in 0.6.1:

```
$ amm -f /dev/null
Loading...
Welcome to the Ammonite Repl 0.6.1
(Scala 2.11.8 Java 1.8.0_92)
@ throw new RuntimeException("wat")
@ throw new IllegalStateException("wat")
```

Squelching any custom predef there just to be sure. `lastException` works, but swallowing errors silently wasn't intended behavior.

**Comment by lihaoyi** [comment]
Yeesh I really need a Beta process to catch this sort of thing...

**Comment by lihaoyi** [comment]
Published 0.6.2 to fix this...
